pkg/ds/daemon_set_test.go

```diff
@@ -302,20 +302,22 @@ func TestSchedule(t *testing.T) {
 	//
 	// Deleting the daemon set should unschedule all of its nodes
 	//
+	// beforeDeletePods := labeledPods(t, ds)
```

Comment:
why leave this commented line in?

edit: oh hmm it gets made into a real line in next commit. oooook